TheMovieDB/TheMovieDBTests/NetworkTests.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  NetworkTests.swift
+//  TheMovieDBTests
+//
+//  Created by Juan Martin Rezk Elso on 23/9/21.
+//
+
+import XCTest
+@testable import TheMovieDB
+
+class NetworkTests: XCTestCase {
+    func testRetrieveMovies() throws {
+        let myExpectation = expectation(description: "task in background")
+        let results = Facade.shared.retrieveData { (results) in
+            switch results {
+            case .success(let movies):
+                XCTAssertNotNil(results)
+                XCTAssertGreaterThan(movies.results.count, 0)
+            case .failure(let error):
+                XCTAssertNotNil(error)
+            }
+            myExpectation.fulfill()
```

Comment:
you should put this before all the other code in the completion because it could fail if it is called later. Put ir in the line 15